x-pack/plugins/osquery/public/live_queries/form/pack_queries_status_table.tsx

```diff
@@ -595,7 +616,18 @@ const PackQueriesStatusTableComponent: React.FC<PackQueriesStatusTableProps> = (
   );
 
   const renderLensResultsAction = useCallback((item) => <PackViewInLensAction item={item} />, []);
+  const handleAddToCase = useCallback(
+    (payload: { actionId: string; isIcon?: boolean }) =>
+      // eslint-disable-next-line react/display-name
+      () => {
+        if (addToCase) {
+          return addToCase({ actionId: payload.actionId, isIcon: payload?.isIcon });
+        }
 
+        return <></>;
```

Comment:
nit: You can return `null`

Reply:
That would be great, but I am afraid it will not fit this particular case. We push the AddToCase Button to a render function in columns (table), which accepts only Element, not null. I think that this <></> is safer than changing the previous type. Is it ok?

Reply:
Oh, I see. Yes, that makes sense.